spec/models/miq_product_feature_spec.rb

```diff
@@ -2,7 +2,7 @@
 require 'pathname'
 
 describe MiqProductFeature do
-  let(:expected_feature_count) { 892 }
+  let(:expected_feature_count) { 892 + 36 }
```

Comment:
Just put `928` in the final version of the PR (I'm sure it's much easier this way as you are changing things :) )